### PR TITLE
Add support for VS 2022

### PR DESCRIPTION
**Pull request by martins-vds**
Added support for Visual Studio 2022

### CODE REVIEW

**Comment by martins-vds** [comment]
@RadoslavGatev @DamienLaw would you guys be able to review this PR please?

File: src/SSDT-CD-VSIX/source.extension.vsixmanifest [hunk 1]
```diff
@@ -11,9 +11,24 @@
     </Metadata>
     <Installation>
         <!-- "[11.0,16.0)" specifies that this extension is deployable to all versions of VS from 11.0 to 15.999.999, but not 16.0  -->
-        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Professional" Version="[15.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,17.0)">
+            <ProductArchitecture>x86</ProductArchitecture>
```
Comment (RadoslavGatev) [inline src/SSDT-CD-VSIX/source.extension.vsixmanifest:15]: Is there any particular reason to specify the architecture? Can't we just use the previous target IDs and just bump up the version?

**Review by RadoslavGatev — COMMENTED** [review]
Hi @martins-vds,
Thanks a lot for taking the time to submit this PR!

I left one question for you. After we settle that, I will need some time to test this out and release the version into the marketplace.

Radoslav

**Comment by RadoslavGatev** [comment]
Thanks again,

I submitted a PR with some additional changes in the template itself.

**Pull request closed without merging**